books/free-programming-books-hi.md

```diff
@@ -7,3 +7,9 @@
 
 * [C++ Brief Notes \| Hindi](https://ehindistudy.com/2020/12/01/cpp-notes-in-hindi/) - Yugal Joshi
 * [C++ Introduction Book \| Hindi](https://ncsmindia.com/wp-content/uploads/2012/04/c++-hindi.pdf) - NCMS India (PDF)
+
+
+### <a id="java"></a>Java
+
+* [Java Brief Notes \| Hindi](https://www.codewithharry.com/videos/java-tutorials-for-beginners-1/) - CodeWithHarry
```

Comment:
Not a book, move it to courses

Reply:
@shrn01 hey sir this website contains the course but also having all the notes in bottom of website and also covers notes in a form of blog 
and sir this is alerady added in courses but  there is youtube link alerady not notess link attached with website

Reply:
it's a duplicate then.